README.md

```diff
@@ -114,14 +114,19 @@ dependency can be provided two ways:
 If you do not wish to build and run the tests, add `-DBENCHMARK_ENABLE_GTEST_TESTS=OFF`
 to `CMAKE_ARGS`.
 
+### Speeding up the build
+
+If you find the build or test runs to be slow, you can pass `--parallel` to the
+`cmake` and `ctest` commands as per the cmake documentation.
```

Comment:
I would rather use `--parallel <jobs>` to make it clear, that there must be some argument.

Also, this is somewhat more complicated, since build can deal without the argument (will default to something) while test cannot. Also, build supports the `--parallel` only since CMake 3.12 (if I recall correctly the check I did - for sure not the required CMake 3.5).

Perhaps it would be nice to actually add links to CMake documentation (cmake(1) and ctest(1) in this case)? Either the required version (3.5) or master version.